src/common.rs

```diff
@@ -981,9 +981,45 @@ pub fn get_current_pid() -> Result<Pid, &'static str> {
     inner()
 }
 
+/// MAC address for network interface
+#[derive(PartialEq, Eq, Copy, Clone, Debug)]
+pub struct MacAddr {
+    data: [u8; 6],
+}
```

Comment:
Please make the field public. Also, what do you think about using a tuple struct instead? Example:

```suggestion
pub struct MacAddr(pub [u8; 6]);
```

Reply:
Ok. Naming an inner field `data` doesn't provide useful information.